test

```diff
@@ -141,7 +141,7 @@ function functional_pass {
 	rm -rf ./agent-*
 
 	if [[ "${ETCD_TESTER_EXIT_CODE}" -ne "0" ]]; then
-		echo "FAIL with exit code" ${ETCD_TESTER_EXIT_CODE}
+		echo "--- FAIL: exit code" ${ETCD_TESTER_EXIT_CODE}
```

Comment:
why adding `---`?

Reply:
To make failure same as go test failures, so that we can match by `egrep`.

Reply:
I wonder if egrep is `egrep "(FAIL|leak)" -A10 -B50 test-${TEST_SUFFIX}.log
View`. Will that also catch previous error msg`echo "FAIL with exit code" ${ETCD_TESTER_EXIT_CODE}`?

just a suggestion. I am fine with adding `---` as well if you think that's the best.

Reply:
`egrep "FAIL"` would fail the test when we enable gRPC client-side logging because now gRPC logs `TRANSIENT_FAILURE` in its balancer wrapper.

ref.
```bash
echo TRANSIENT_FAILURE | egrep "FAIL"
```

Reply:
I see. Thanks for the clarification.